gaffer-as-a-service/gaas-jwt-sidecar/src/main/java/uk/gov/gchq/gaffer/sidecar/exception/GaaSRestApiException.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.sidecar.exception;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+public class GaaSRestApiException extends Exception {
```

Comment:
Isn't this the same as [GaaSRestApiException](https://github.com/gchq/gaffer-experimental/blob/a0c0f36267f1774b8e117fd8e33b7fc3ce15b063/gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/exception/GaaSRestApiException.java)?

Reply:
Yes will refactor to include dependency on rest.

Reply:
In fact currently unneeded so have removed